helm/cluster-openstack/ci/ci-values.yaml

```diff
@@ -33,14 +33,16 @@ controlPlane:
   bootFromVolume: true
 
 nodeClasses:
-- diskSize: 30
-  image: worker-image
-  flavor: worker-flavor
-  name: worker
-  bootFromVolume: true
+  default:
+    diskSize: 30
+    image: worker-image
+    flavor: worker-flavor
+    name: default
```

Comment:
would name still be needed here?

Reply:
I wanted to keep it for compatibility issues, so you can still access as property

Reply:
I'd strongly prefer to remove it. It may be different from the key and we don't have any validation in place to ensure that.

Reply:
In the last commit, it is deleted. See https://github.com/giantswarm/cluster-openstack/pull/120#issuecomment-1286767364

Reply:
I still see it